ibis/backends/dask/tests/test_udf.py

```diff
@@ -259,7 +259,7 @@ def test_udaf_analytic_groupby(con, t, df):
     def f(s):
         return s.sub(s.mean()).div(s.std())
 
-    expected = df.groupby('key').c.transform(f)
+    expected = df.groupby('key').c.transform(f).compute()
```

Comment:
what did this do before?

Reply:
Err this was/is still being skipped because the underlying functionality is `NotImplemented`. But the `compute` will need to be there anyway eventually (I think I forgot to fix this when we changed `.execute()` to return a computed object.